paramak/parametric_reactors/ball_reactor.py

```diff
@@ -182,7 +189,7 @@ def _make_plasma(self):
         )
         plasma.create_solid()
```

Comment:
Is this useful ?

Reply:
I guess plasma.create_solid() can be removed

Reply:
The reason plasma.create_solid is there is because the vertical build need to access the high point of the plasma

Reply:
I see. Since these points don't need the whole create_solid method to be called (only plasma parameters), how about we create setters/getters for the high points, equatorial points, etc ?

Reply:
that sounds like the way to go